### PR TITLE
Revert "disables smartgunner, makes smart machinegun useable by anyone"

### PR DESCRIPTION
**Pull request by Pariah919**
Reverts tgstation/TerraGov-Marine-Corps#6241
this wasn't a good idea
might keep the sg being useable by anyone but whatever

### CODE REVIEW

**Comment by TiviPlus** [comment]
No format and hasnt been a month like spooky said

File: code/modules/projectiles/guns/rifles.dm [hunk 1]
```diff
@@ -889,6 +889,7 @@
 	gun_iff_signal = list(ACCESS_IFF_MARINE)
 	gun_firemode_list = list(GUN_FIREMODE_SEMIAUTO, GUN_FIREMODE_AUTOMATIC)
 	starting_attachment_types = list(/obj/item/attachable/stock/t29stock, /obj/item/attachable/t29barrel)
+	gun_skill_category = GUN_SKILL_SMARTGUN //Uses SG skill for the penalties.
```
Comment (Pariah919) [inline code/modules/projectiles/guns/rifles.dm:892]:
```suggestion
```

**Comment by DerrickChamberlain** [comment]
Damn closed already?

**Comment by DerrickChamberlain** [comment]
We all know it, stop hiding from the truth.

**Comment by DerrickChamberlain** [comment]
a

> Reverts #6241
> this wasn't a good idea
> might keep the sg being useable by anyone but whatever

Keeping the class and the skill kept the weapon balanced. We must return to greatness.

**Pull request closed without merging**